cdc/model/changefeed_test.go

```diff
@@ -190,6 +190,30 @@ func (s *configSuite) TestVerifyAndFix(c *check.C) {
 	c.Assert(marshalConfig1, check.Equals, marshalConfig2)
 }
 
+func (s *configSuite) TestChangeFeedInfoClone(c *check.C) {
+	defer testleak.AfterTest(c)()
+	info := &ChangeFeedInfo{
+		SinkURI: "blackhole://",
+		Opts:    map[string]string{},
```

Comment:
i remember a empty map will become nil in some case. will it break something?

Reply:
Not the same thing, `map[string]string{}` or `new(map[string]string)` has initialized the map, it is not nil anymore

Reply:
I checked that only for Unmarshal of `protoc-gen-gogo`, empty map will become nil 😂